tests/acceptance/features/webUIRenameFolders/renameFolders.feature

```diff
@@ -58,56 +58,45 @@ Feature: rename folders
     And the user reloads the current page of the webUI
     Then folder "  multiple   spaces    all     over   " should be listed on the webUI
 
-  @skip
-  Scenario: Rename a folder using both double and single quotes
-    When the user renames the following folder using the webUI
-      | from-name-parts | to-name-parts         |
-      | simple-folder   | First 'single' quotes |
-      |                 | -then "double"        |
+  Scenario Outline: Rename a folder using both double and single quotes
+    When the user renames folder <from_name> to <to_name> using the webUI
     And the user reloads the current page of the webUI
-    Then the following folder should be listed on the webUI
-      | name-parts            |
-      | First 'single' quotes |
-      | -then "double"        |
-    When the user renames the following folder using the webUI
-      | from-name-parts       | to-name-parts   |
-      | First 'single' quotes | a normal folder |
-      | -then "double"        |                 |
+    Then folder <to_name> should be listed on the webUI
+    When the user renames folder <to_name> to <next_name> using the webUI
     And the user reloads the current page of the webUI
-    Then folder "a normal folder" should be listed on the webUI
+    Then folder <next_name> should be listed on the webUI
+    Examples:
+      | from_name       | to_name                 | next_name         |
+      | "simple-folder" | "First 'single' quotes" | "a normal folder" |
```

Comment:
this will only use the single quotes, we want to have tests that have both single and double quote in a single file name

Reply:
:guitar: